service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FailedPBAPaymentEventHandler.java

```diff
@@ -27,32 +36,72 @@
     private final FailedPBAPaymentContentProvider notificationContent;
 
     @EventListener
-    public void notifyLocalAuthority(FailedPBAPaymentEvent event) {
+    public void notifyApplicant(FailedPBAPaymentEvent event) {
+        CaseData caseData = event.getCaseData();
+        String applicant = defaultIfNull(event.getApplicantName(), "");
+        if (event.getApplicationTypes().contains(ApplicationType.C110A_APPLICATION)) {
+            notifyLocalAuthority(event, APPLICATION_PBA_PAYMENT_FAILED_TEMPLATE_FOR_LA);
+        } else {
+            if ((caseData.getCaseLocalAuthorityName() + ", Applicant").equals(applicant)) {
+                notifyLocalAuthority(event, INTERLOCUTORY_PBA_PAYMENT_FAILED_TEMPLATE_FOR_APPLICANT);
+            } else {
+                Map<String, Collection<String>> emails = getRespondentsEmails(caseData);
+                if (isNotEmpty(emails.get(applicant))) {
+                    notifyRespondent(event, emails.get(applicant));
+                }
+            }
+        }
+    }
+
+    private void notifyLocalAuthority(FailedPBAPaymentEvent event, String template) {
         CaseData caseData = event.getCaseData();
 
         FailedPBANotificationData parameters = notificationContent
-            .getLocalAuthorityNotifyData(event.getApplicationType());
+            .getApplicantNotifyData(event.getApplicationTypes(), event.getCaseData().getId());
 
         Collection<String> emails = inboxLookupService.getRecipients(
             LocalAuthorityInboxRecipientsRequest.builder()
                 .excludeLegalRepresentatives(true)
                 .caseData(caseData)
                 .build());
 
-        notificationService.sendEmail(APPLICATION_PBA_PAYMENT_FAILED_TEMPLATE_FOR_LA, emails, parameters,
-            caseData.getId().toString());
+        notificationService.sendEmail(template, emails, parameters, caseData.getId().toString());
+    }
+
+    private void notifyRespondent(FailedPBAPaymentEvent event, Collection<String> emails) {
+        FailedPBANotificationData parameters = notificationContent
+            .getApplicantNotifyData(event.getApplicationTypes(), event.getCaseData().getId());
+
+        notificationService.sendEmail(INTERLOCUTORY_PBA_PAYMENT_FAILED_TEMPLATE_FOR_APPLICANT,
+            emails, parameters, event.getCaseData().getId().toString());
+    }
+
+    private Map<String, Collection<String>> getRespondentsEmails(CaseData caseData) {
+        Map<String, Collection<String>> respondentEmails = new HashMap<>();
+
+        IncrementalInteger i = new IncrementalInteger(1);
+        caseData.getAllRespondents().forEach(respondent -> respondentEmails.put(
+            respondent.getValue().getParty().getFullName() + ", Respondent " + i.getAndIncrement(),
+            Set.of(respondent.getValue().getSolicitor().getEmail())));
```

Comment:
this may end with NPE, not every respondent have solicitor. Beside wrapping into Set suggest there can be multiple recipients